doc/users/credits.rst

```diff
@@ -178,45 +313,81 @@ Jochen Voss,
 Jody Klymak,
 Joe Kington,
 Joel B. Mohler,
+Johannes Wienke,
+John Hoffman,
 John Hunter,
+John Vandenberg,
+Johnny Gill,
+JojoBoulix,
 Jonathan Waltman,
 Jorrit Wronski,
 Josef Heinen,
+Joseph Albert,
+Joseph Fox-Rabinovitz,
 Joseph Jon Booker,
+Joseph Martinot-Lagarde,
 José Ricardo,
 Jouni K. Seppänen,
+Juan Nunez-Iglesias,
+Julia Sprenger,
 Julian Mehne,
 Julian Taylor,
+Julian V. Modesto,
 JulianCienfuegos,
+Julien L,
```

Comment:
That's Julien Lhermitte (just below), see https://github.com/matplotlib/matplotlib/pull/8455/commits.